src/regressionmodel.jl

```diff
@@ -132,3 +132,43 @@ function linearpredictor end
 In-place version of [`linearpredictor`](@ref), storing the result in `storage`.
 """
 function linearpredictor! end
+
+"""
+    vif(m::RegressionModel)
+
+Compute the variance inflation factor (VIF).
+
+Returns a vector of inflation factors computed for each coefficient except
+for the intercept.
+In other words, the corresponding coefficients are `coefnames(m)[2:end]`.
+
+The [variance inflation factor (VIF)](https://en.wikipedia.org/wiki/Variance_inflation_factor) measures
```

Comment:
```suggestion
The [VIF](https://en.wikipedia.org/wiki/Variance_inflation_factor) measures
```
You've defined VIF and provided the acronym above, so you can reference it by the acronym here.